app/models/concerns/pageflow/uploaded_file.rb

```diff
@@ -55,5 +55,13 @@ def cache_key
       # prevent caching outdated information.
       "#{super}-#{state}"
     end
+
+    def direct_upload_config
+      presigned_post_config = Pageflow.config.paperclip_direct_upload_options.call(attachment)
+      {
+        url: presigned_post_config.url,
+        fields: presigned_post_config.fields
+      }
+    end
```

Comment:
Let's change `Pageflow.config.paperclip_direct_upload_options` to return this hash directly. This way we can avoid the `OpenStruct` in the test stub.